src/RoboFile/RoboFileInterface.php

```diff
@@ -43,4 +43,26 @@ public function runTestSuites(
 	 * @since   3.7.0
 	 */
 	public function prepareTestingPackage($params = array('dev' => false));
+	/**
+	 * Function for actual execution of a individual test of this extension
+	 *
+	 * @param   array  $opts  Array of configuration options:
+	 *                        - 'env': set a specific environment to get configuration from
+	 *                        - 'debug': executes codeception tasks with extended debug
+	 * 						  - 'suite':
+	 * 						  - 'test': example: 'install/InstallWeblinksCest.php/installWeblinks'
+	 *
+	 * @return void
+	 *
+	 * @since   x.x.x
+	 */
+	public function runTestSingle(
```

Comment:
well, remember that in the final environment, the main coordinator will know which task it's executing from weblinks, and assigning it to the container, so there should not be a need for any actual implementation of the function in weblinks, since the call will be just the same for every particular test (in terms of executed code)